src/js/tech/html5.js

```diff
@@ -200,6 +200,14 @@ class Html5 extends Tech {
     });
   }
 
+  overrideNativeAudioTracks(override) {
```

Comment:
Needs a jsDoc explaining the intent of these functions and how it will take affect on source change